consul-hcs-azure/step1.md

```diff
@@ -0,0 +1,41 @@
+### Log into the Azure CLI
+
+First, you need to login to the Azure CLI using the login
```

Comment:
```suggestion
First, you need to login to the Azure CLI using your credentials or the login
```
(If this will be used after the class by the general public)

Reply:
> (If this will be used after the class by the general public)

I don't know that it will be, and it does require pre-provisioning of HCP and AKS clusters, so I'm going to delay this for now